edx/blockstore
# Remove tagstore.

## Description

Tagstore is not being used so the code is being removed. Systems using blockstore for content storage should develop their own tagging system appropriate for their needs.

## Test Instructions

The tagstore code is not being used anywhere. Running the end-to-end tests is sufficient to confirm that the bundles code is working as expected.

To test the migrations:
```
git checkout master
docker exec -t edx.devstack.mysql57 /bin/bash -c 'mysql -uroot <<< "drop database blockstore_db;"'
docker exec -t edx.devstack.mysql57 /bin/bash -c 'mysql -uroot <<< "create database if not exists blockstore_db;"'
make blockstore-shell
./manage migrate
./manage dbshell
mysql> show tables; # This will print all tables including `tagstore_entity`, `tagstore_entity_tags`, `tagstore_tag`, `tagstore_taxonomy`.
exit
exit

git checkout symbolist/remove-tagstore
make blockstore-shell
./manage.py migrate # This will apply tagstore_django.0002_delete_all_app_models.
./manage dbshell
mysql> show tables; # The tagstore_ tables will no longer show in the list.
```
